.github/workflows/release-packages.yml

```diff
@@ -46,3 +46,32 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  sync_branches:
+    name: "Sync `main` with `canary`"
+    runs-on: ubuntu-latest
+    needs: release_packages
+    if: ${{ needs.release_packages.outputs.published == 'true' }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Create PR
+        uses: TreTuna/sync-branches@1.4.0
+        with:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          FROM_BRANCH: "canary"
+          TO_BRANCH: "main"
+          PULL_REQUEST_TITLE: Deploy faustjs.org and example projects
+          PULL_REQUEST_BODY: |
+            New releases just landed in `canary`, so let's bring `main` up to speed!
```

Comment:
Is it possible here to interpolate the [**FROM_BRANCH**](https://github.com/marketplace/actions/sync-branches#from_branch) and **TO_BRANCH** variables?

Reply:
I wish, but unfortunately the action doesn't support that.